README.md

```diff
@@ -20,14 +20,13 @@ npm i undici
 
 ## Benchmarks
 
-Machine: Intel i7-7700k, 4.5GHz, 4C/8T, 32GB RAM, NVMe SSD\
-Server: [H2O](https://github.com/h2o/h2o) 2.2.5\
-Configuration: HTTP/1.1 without TLS
+Machine: 2.7 GHz Quad-Core Intel Core i7
```

Comment:
Can the benchmarks be run on GitHub Actions? Just asking... :)

Reply:
I wonder how would be the performance there...

Reply:
I have no idea how to do that... sounds like a good idea though